tests/commands/test_configure.py

```diff
@@ -1,59 +1,59 @@
-from collections import namedtuple
```

Comment:
@pm3310 Missed this when reviewing. Did these tests need to be disabled? Can we re-enable them? Also, looks like the 0.20.0 merge failed on _Python3_.

Reply:
@ilazakis good catch man! My bad :-( I'll create a new PR for this one